addons/bestja_frontend_fpbz/__openerp__.py

```diff
@@ -0,0 +1,34 @@
+{
+    # Theme information
+    'name': "FPBŻ Theme",
+    'description': """
```

Comment:
We use both "description" and "summary" (that you omitted). If I remember correctly Odoo throws some warnings otherwise.